### DISCUSSION TITLE
PClub Summer of Code : Participants Query

### DISCUSSION THREAD

**Discussion by infiniteoverflow** [infiniteoverflow/Libro#138] [General]
This discussion channel is for PClub Summer of Code Participants who would like to get started with Libro

**Comment by rs7358** [comment]
Hi, I'm very interested behind the idea of this app. It's something I'd like to have as an app on my phone :)
Could you let me know what specific task is to be completed with regards to PClub Summer of Code? I'm new to Flutter Dev so any clarity and guidance will be helpful for me to contribute to this project and learn alongside.

**Comment by apoorva0107** [comment]
Hi, I would like to participate in the project. I recently developed an interest in Android development and currently learning more while working on my personal project. I would like to be a part of this project and expand my vision of Android development and learn from the community. Please guide me on what to include in the proposal. Thanks!

**Comment by infiniteoverflow** [comment]
So, welcome @ronit-singh and @apoorva0107 to Project Libro !! I am happy to see your interest in the app. So i will explain below what all are my visions about the project by the end of this Summer of Code event.

1. We need to make sure all the authentication flow works properly, and once a user is authenticated , she/he has to verify their email and then proceed onto a screen where the user can enter their details and even upload their pic (optional to the user).

2. Then these details has to be stored in the database (preferably FirebaseRealtimeDatabase) with proper structure so that we can retrieve them efficiently.

3. We need to make sure the profile screen of the app is upto mark and should contain the details of the user.

4. We need to have an option to upload a book (will explain the workflow in some days).

So basically this is my plan for this month , if we catch up with these soon, we can move onto other features.


**Concepts in Flutter i believe you will be using : Firebase , Providers**
